lib/gemstash/gem_source/upstream_source.rb

```diff
@@ -94,7 +94,7 @@ def serve_prerelease_specs
 
       def upstream
         @upstream ||= Gemstash::Upstream.new(env["gemstash.upstream"],
-          user_agent: env["gemstash.user-agent"])
+                                             user_agent: env["gemstash.user-agent"])
```

Comment:
what about:
```ruby
@upstream ||= Gemstash::Upstream.new(
  env["gemstash.upstream"],
  user_agent: env["gemstash.user-agent"]
)

```
?

Reply:
If this doesn't break the rules in place, please make the change!

Reply:
It didn't break the rules, I made the change, and rebaseed.